src/test/java/de/tum/in/www1/artemis/exercise/programmingexercise/ProgrammingExerciseTest.java

```diff
@@ -1,11 +1,10 @@
 package de.tum.in.www1.artemis.exercise.programmingexercise;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
 
 import java.time.ZonedDateTime;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
are you sure that you import >4 classes from `java.util`?

Reply:
It is =4, I added Optional. However, due to `ij_java_class_count_to_use_import_on_demand = 4` in the .editorconfig file, the * import is used.